src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -101,7 +101,9 @@ export class DashboardComponent implements OnInit, OnDestroy {
         for (const graph of this.selectedGraphsList) {
           const selectedGraph = this.statistics.filter(value => value['key'] === graph.key);
```

Comment:
Can't we use `find` instead of `filter`? here because there will be only one returned value and `filter` always return array. that is why we have to use  `selectedGraph[0]`. I think we can eliminate that by using find?